go.mod

```diff
@@ -101,22 +101,24 @@ require (
 	github.com/skratchdot/open-golang v0.0.0-20200116055534-eef842397966
 	github.com/slack-go/slack v0.6.5
 	github.com/stretchr/testify v1.7.0
+	github.com/vdemeester/k8s-pkg-credentialprovider v1.18.1-0.20201019120933-f1d16962a4db // indirect
```

Comment:
Hey! I still see a bunch of k8s things in here 🙈 sorry to be a pain, I want to make sure your contribution is added 😅